framework_tvb/tvb/adapters/analyzers/bct_adapters.py

```diff
@@ -207,36 +214,39 @@ class DistanceDBIN(BaseBCT):
     """
     _ui_group = BCT_GROUP_DISTANCE
 
-    _ui_name = "Distance binary matrix"
+    _ui_name = "Distance binary matrix: Connection matrix"
     _ui_description = bct_description("distance_bin.m")
     _matlab_code = "D = distance_bin(A);"
 
-    def launch(self, connectivity, **kwargs):
-        kwargs['A'] = connectivity.weights
-        result = self.execute_matlab(self._matlab_code, **kwargs)
+    def launch(self, view_model):
+        connectivity = self.get_connectivity(view_model)
+        data = {'A': connectivity.weights}
+
+        result = self.execute_matlab(self._matlab_code, data=data)
         measure = self.build_connectivity_measure(result, 'D', connectivity, "Distance matrix")
         return [measure]
 
 
 class DistanceDWEI(DistanceDBIN):
     """
     """
-    _ui_connectivity_label = "Weighted (directed/undirected) connection matrix:"
-    _ui_name = "Distance weighted matrix"
+    _ui_name = "Distance weighted matrix: Weighted (directed/undirected) connection matrix"
```

Comment:
Similarly, all the ui_names could get a review, to make more sense
`Distance weighted matrix over a Weighted (directed/undirected) connection matrix`